src/list/manage/components/list-sort.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import classNames from "../../../common";
+
+import styles from "./list-sort.css";
+
+export default class ListSort extends React.Component {
+  // TODO: do we need to do something with className?
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.value,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(evt) {
+    const value = evt.target.value;
+    this.setState({ value });
+    // TODO: onChange vs handleChange is confusing
+    this.props.onChange(value);
+  }
+
+  render() {
+    const { onChange, sort, className, disabled, ...props } = this.props;
```

Comment:
Per earlier comment that onChange is confusing, I don't think you need onChange destructured here 😁 Same for className